spyder/plugins/variableexplorer/widgets/namespacebrowser.py

```diff
@@ -73,6 +73,13 @@ def __init__(self, parent):
         self.editor = None
         self.shellwidget = None
         self.finder = None
+        self.pane_empty = PaneEmptyWidget(
+            self,
+            "variable-explorer",
+            _("No variables defined yet"),
```

Comment:
```suggestion
            _("No variables to show"),
```

Likewise, if the user `del`s all visible variables, clears variables in their kernel, restarts it, switches to another with no variables defined, etc., then this message would be incorrect and misleading. Furthermore, this would also be shown if there were variables defined (as there always are some) but they are not shown due to the filters, which exclude many variable types by default, but it could clearly confuse the user into thinking they hadn't defined any.

Therefore, the suggested revision fixes both these issues, making it clear that there are simply no variables the pane has to show currently as opposed to that the user has never defined any, using consistent terminology from other panes and UIs.